assets/sass/components/user-input/googlesitekit-user-input-preview.scss

```diff
@@ -144,4 +144,12 @@
 			}
 		}
 	}
+
+	.googlesitekit-user-input__preview-group .googlesitekit-user-input__select-option .mdc-radio {
```

Comment:
@nfmohit Let's simplify those lines into the following:
```css
.googlesitekit-user-input__preview-group {
    .googlesitekit-user-input__select-option {
        .mdc-radio {
            margin-left: -10px;
        }

        &:not(:last-child) {
            border-bottom: none;
        }
    }
}
```

Reply:
Updated, thank you @asvinb!